dev/AppLifecycle/ExtensionContract.h

```diff
@@ -39,7 +42,7 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
                 for (const auto& extension : c_extensionMap)
                 {
                     std::wstring contractId = pair.Value().c_str();
-                    if (CompareStringOrdinal(contractId.c_str(), -1, extension.contractId.c_str(), -1, TRUE) == CSTR_EQUAL)
+                    if (CompareStringOrdinal(contractId.c_str(), -1, extension.contractId, -1, TRUE) == CSTR_EQUAL)
```

Comment:
>CompareStringOrdinal(contractId.c_str(), -1, extension.contractId, -1, TRUE) == CSTR_EQUAL [](http://example.com/codeflow?start=24&length=90)

We seem to be writing `CompareStringOrdinal(X, -1, Y, -1, TRUE) == CSTR_EQUAL` a lot. Maybe factor into helper function.

```cpp
bool IsEqualOrdinalCaseInsensitive(PCWSTR a, PCWSTR b)
{
return CompareStringOrdinal(a, -1, b, -1, TRUE) == CSTR_EQUAL;
}
```